docroot/modules/custom/nvli_custom_search/src/Form/CustomSearchForm.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\nvli_custom_search\Form\CustomSearchForm.
+ */
+
```

Comment:
As per Drupal coding standard https://www.drupal.org/coding-standards/docs

@file tag docblock should not be there in the files that contain a namespaced class/interface/trait, whose file name is the class name with a .php extension.